webpack.config.server.js

```diff
@@ -0,0 +1,14 @@
+const StartServerPlugin = require('start-server-webpack-plugin');
+
+module.exports = ({ resolvePath, START_SERVER }) => {
+  return {
+    target: 'node', // compile for server environment
+    entry: ['./src'],
+    output: {
+      path: resolvePath('build'),
+      filename: 'server.js',
+    },
+    // only start the server if we've run `npm run dev`
```

Comment:
I'm not sure I follow this comment? Why would this not be true for `npm start` on a server?

Reply:
Hmm what I meant was `npm run build` should _not_ start the server 👍 

If we didn't have any conditional logic and we just run `webpack`, it would run all of the plugins (including the server). We only want it triggered from certain commands.

I'll go back and tweak the comment to make more sense :)